pom.xml

```diff
@@ -11,7 +11,7 @@
 	<groupId>com.entropy</groupId>
 	<artifactId>entropay.employees</artifactId>
```

Comment:
entropay looks more like part of the groupId? I'd use `-` instead of `.`

Reply:
on top of what @umarcostuc said, we should use the domain of the company. Therefore, please use `com.entropyteam.entropay` as the `groupId`